--- Perlbal-1.59/t/15-webserver.t.orig	2007-09-16 18:52:34.661705909 +0100
+++ Perlbal-1.59/t/15-webserver.t	2007-09-16 18:53:07.656672320 +0100
@@ -64,7 +64,8 @@
 ok(get("$url?foo=bar") eq $contents, "GET request");
 
 {
-    my $req = HTTP::Request->new(GET => $url, [ 'If-Modified-Since' => HTTP::Date::time2str() ]);
+    my $ims = HTTP::Date::time2str( ( stat $disk_file )[9] );
+    my $req = HTTP::Request->new(GET => $url, [ 'If-Modified-Since' => $ims ]);
     my $res = $ua->request($req);
 
     is($res->code, 304, "Got not modified");
